content/post/Matter_future.md

```diff
@@ -0,0 +1,127 @@
++++
+title = 'Matter and the future'
+slug = 'Matter_future'
+image = '/static/images/Cover.jpeg'
+date = "2023-03-23T10:00:00"
+author = 'Tushar Bhatia'
+tags=['IoT', 'Networking', 'Smart Home', 'Smart Devices', 'Google-projects']
+categories =['IoT', 'Smart-Devices', 'Smart Homes', 'Networks']
+description = 'This blog explores Matter a new communication protocol made by CSA'
+disableComments = false
++++
+
+Hi everyone, This blog explores the new communication protocol, known as Matter which is being developed by <strong>Connectivity Standards Alliance (CSA)</strong>
+<br>
+To know more, Let's go on a journey and explore the new generation of communication protocols.
+<br>
+
```

Comment:
```suggestion
Hi everyone, this blog explores the new communication protocol, known as Matter which is being developed by **Connectivity Standards Alliance (CSA)**

To know more, Let's go on a journey and explore the new generation of communication protocols.
```

Please replace HTML tags with plain markdown, here's an example of the conversion.